gentest/fixtures/YGAlignContentTest.html

```diff
@@ -29,3 +29,23 @@
   <div style="width: 50px;"></div>
   <div style="width: 50px;"></div>
 </div>
+
+<div id="align_content_stretch_row" style="width: 150px; height: 100px; flex-wrap: wrap; flex-direction: row; align-content: stretch;">
+  <div style="width: 50px;">
+    <div style="flex: 1"></div>
+  </div>
+  <div style="width: 50px;flex: 1;"></div>
```

Comment:
I wasn't sure if the layout is correctly set already if the node is a flex node. It changed the implementation during development.